Testing/functional/tests/cypress/integration/e2e/user/dependents/dashboard.js

```diff
@@ -0,0 +1,408 @@
+const validDependentCovid19TestResultsButtonId = `[data-testid=dependent-entry-type-Covid19TestResult-${validDependentHdid}]`;
+const validDependentFederalProofOfVaccinationButtonId = `[data-testid=proof-vaccination-card-btn-${validDependentHdid}]`;
+const validDependentImmunizationsButtonId = `[data-testid=dependent-entry-type-Immunization-${validDependentHdid}]`;
+const validDependentLabResultsButtonId = `[data-testid=dependent-entry-type-LabResult-${validDependentHdid}]`;
+
+function validateDashboardCard(buttonId, filterType) {
+    cy.get(buttonId).should("be.enabled", "be.visible").click();
+    cy.location("pathname").should("eq", validDependentTimelinePath);
+    cy.checkTimelineHasLoaded();
+
+    cy.get("[data-testid=filterContainer]").should("not.exist");
+    cy.get("[data-testid=filterDropdown]").click();
+    cy.get(`[data-testid=${filterType}-filter]`).should("to.be.checked");
+    cy.get("[data-testid=btnFilterCancel]").click();
+}
+
+describe("dependents - dashboard", () => {
+    beforeEach(() => {
+        cy.configureSettings({
+            homepage: {
+                showFederalProofOfVaccination: true,
+            },
+            dependents: {
+                enabled: true,
+                timelineEnabled: true,
+            },
+            datasets: [
+                {
+                    name: "immunization",
+                    enabled: true,
+                },
+                {
+                    name: "covid19TestResult",
+                    enabled: true,
+                },
+                {
+                    name: "clinicalDocument",
+                    enabled: true,
+                },
+                {
+                    name: "labResult",
+                    enabled: true,
+                },
+            ],
+        });
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak,
+            "/dependents"
+        );
+    });
+
+    it("Validate dashboard immunizations tab click to timeline", () => {
+        validateDashboardCard(
+            validDependentImmunizationsButtonId,
+            "Immunization"
+        );
+    });
+
+    it("Validate dashboard lab results tab click to timeline", () => {
+        validateDashboardCard(validDependentLabResultsButtonId, "LabResult");
+    });
+
+    it("Validate dashboard covid19 test results tab click to timeline", () => {
+        validateDashboardCard(
+            validDependentCovid19TestResultsButtonId,
+            "Covid19TestResult"
+        );
+    });
+
+    it("Validate dashboard clinical documents tab click to timeline", () => {
+        validateDashboardCard(
+            validDependentClinicalDocumentsButtonId,
+            "ClinicalDocument"
+        );
+    });
+
+    it("Validate dashboard aged out dependent and remove", () => {
```

Comment:
Shouldn't this test check removal via the "Remove Dependent" button that appears for aged-out dependents, instead of via the Delete button in the menu, since that's tested elsewhere?

Reply:
Fixed